### ISSUE TITLE
Crate opening player data // milestone count resets on plugin update

### ISSUE THREAD

**Issue by Projecki** [nulli0n/ExcellentCrates-spigot#194]
I updated ExcellentCrates from version 5.0.0 to 5.2.3.1

This caused all my player's %excellentcrates_openings_<crate_id>% numbers to be set back to 0

Now, players are able to re-earn milestone rewards, which is breaking my economy

Is there any fix for this? How can we properly update the plugin and retain this data?

**Comment by Projecki** [comment]
After testing, I have found with a fresh 5.0.0.0 config being upgraded to 5.2.3.1 does not wipe milestone progress

However, when using my config/crate data, upgrading from 5.0.0.0 to 5.2.3.1 does wipe milestone progress.

I'm unsure why - I tested on a dev server with minimal plugins installed.

Is there anything I can do to safely upgrade ExcellentCrates? What could I change in my config?

Any help is appreciated. I would love to update the plugin. It is a really good plugin and I like the new updates! 

https://youtu.be/7pSK5KKv7mQ
